Core1Platform/BasicPhilosophy/04_ModuleOverview.md

```diff
@@ -0,0 +1,115 @@
+
+**Currency**   
+The *Currency* plug-in provides a list of currencies with their ISO codes as well as daily exchange rates provided by the European Central Bank (ECB).
+
+**Taxes**   
+The *Taxes* module allows you to configure the automatic tax feature of the system to match your specific business requirements. You can define specific tax classes applicable to your products and determine different tax zones where you operate. Additionally, you can set up an unlimited number of tax rates, which you can also specify according to their validity period.   
+For each tax zone &frasl; tax class combination you can define tax rates. You can determine relevant tax zones based on country, ZIP-code, B2B/B2C, and much more.
+
+**Countries and languages**   
+The *Countries and languages* module <!---ist das ein Modul?--> provides any country with their ISO code and supports country classifications such as European Union (EU).
+
+
+**Data hub**   
+The *DataHub* module offers you the possibility to extend any data model with user-defined custom fields. The integrated ETL function allows to control the data flow between the different data models. The most Actindo modules and plugins are based on the *DataHub* module.  
+This enables you to customize your individual data fields and sources for each process. The *DataHub* module is the system’s central data hub and is extended by the following modules:   
+- DataHub Importer  
+Allows the import of any data from various data sources into DataHub-based entities by uploading csv, json, xml, and edi file types via the network protocols HTTP and SFTP.
+
+- DataHub Exporter  
+Allows the export of any data from various data sources into DataHub-based entities by uploading csv, json, xml, and edi  file types via the network protocols HTTP and SFTP.
+
+
+
+## Product data
+
+**Cloudinary**  
+The *Cloudinary* module is an integration of Cloudinary&trade; to Actindo. It allows to output the Cloudinary&trade; images via their URLs in web shops or marketplaces via the ETL mapping in Actindo. It is integrated via the *DataHub* module.  
+ You can use the integration to upload images to Cloudinary&trade; via Actindo, to import images from Cloudinary&trade; to Actindo, and to delete images in Cloudinary&trade; via Actindo.
+
+
+**PIM**  
+The *Product Information Management* (PIM) module is the master data store for collecting, managing, and enriching product data and thus the heart of all your sales channels. You can integrate the *PIM* module quickly into your existing system landscapes. It serves as the basis for efficient unified commerce across all channels.
+The PIM master data are used to provide material master data as well as data needed for the communication with web shops and marketplaces.
```

Comment:
data are / is ? Beide sind möglich, aber in Doku (auch wenn nur pro Modul) konsistent verwenden -> vgl. nächsten Absatz: This master data does not contain... aber The Warehouse management enriches these data...

Reply:
d